tests/unit/components/sl-date-time-test.js

```diff
@@ -16,15 +16,28 @@ test( 'Default class names are present', function( assert ) {
 
 test( 'Attribute "datetime" is properly set', function( assert ) {
     this.subject({
-        timezone: 'America/Chicago',
-        value: new Date( 2015, 0, 1 )
+        format: 'datetime',
+        timezone: 'America/Denver',
```

Comment:
If line 21 is the real fix, then why the other changes?  Specifically, why the timezone change?